cronos-play/getting-started_unreal/quick-start/connect-defi-wallet-with-uri.md

```diff
@@ -0,0 +1,551 @@
+# Connect Defi Wallet with URI
+
+Connecting with Defi Desktop Wallet or Defi Mobile Wallet is accomplished with just one blueprint function: `ConnectWalletConnect`.&#x20;
+
+To connect with QR Code, please check
+
+{% content-ref url="connect-wallets-with-qr-code.md" %}
+[connect-wallets-with-qr-code.md](connect-wallets-with-qr-code.md)
+{% endcontent-ref %}
+
+## ConnectWalletConnect
+
+Once the session is created, the session information would be cached locally, so that we can retrieve the session information even if we restart the game.
+
+{% tabs %}
+{% tab title="ConnectWalletConnect" %}
+#### Source Code
+
+```
```

Comment:
those tabs look good, but it may be good to have a tooltip or a link to some explanation for the "source code", i.e. how one can import it to the Unreal Editor (just copy, open the blueprint window, and paste it?)

Reply:
Added.

Reply:
Gitbook needs time to sync back to github, anyway.

Reply:
Refreshed.